src/main/gen_malli_kondo_config/core.cljs

```diff
@@ -0,0 +1,12 @@
+(ns gen-malli-kondo-config.core
```

Comment:
Since these are dev namespaces they shouldn't be in a production source path. I'll move these over to src/dev-cljs/ . One day it'd be great if we had a dev clj alias as well